pages.it/common/git-fetch.md

```diff
@@ -0,0 +1,24 @@
+# git fetch
+
+> Scarica oggetti e riferimenti da un repository remoto.
```

Comment:
Fetch should translate to 'recupera'. I propose 'scarica' because closer to the actual action